dask_cudf/tests/test_core.py

```diff
@@ -115,6 +115,23 @@ def test_set_index(nelem):
         dd.assert_eq(expect, got, check_index=False, check_divisions=False)
 
 
+def test_timeseries_index():
+
+    df = pd.DataFrame()
+    df['date'] = pd.date_range('11/20/2018', periods=72, freq='D')
+    df['value'] = np.random.sample(len(df))
+
+    gdf = cudf.DataFrame.from_pandas(df)
+    ddf = dgd.from_cudf(gdf, npartitions=2)
```

Comment:
```suggestion
    ddf = dd.from_pandas(gdf, npartitions=2)
```

Reply:
I suggest (though am open to counter-arguments) that we start encouraging the use of the dask.dataframe API when available over dask-cudf

Reply:
👍